src/DropdownMenu/menu.scss

```diff
@@ -1,14 +1,19 @@
-@import '../utils/theme/variables.scss';
+$background: white !default;
+$box-shadow-color: #628292 !default;             // $lynch
+$menu-title-color: #bdc2ca !default;             // $silver
```

Comment:
also here, you could rename "menu-title-color" in "title-color".

(I'm not sure if "menu-item-..." should be renamed to "item-..." I'll leave the decision to you)